cli.js

```
@@ -49,21 +49,19 @@ function action(paths, options) {
   }
 
   if (files.length === 0) {
-    console.error('No files were found.');
-    process.exit(1);
+    throw new Error('No files were found.');
```

We should throw an error instead of console.error